python/chronos/src/bigdl/chronos/data/tsdataset.py

```diff
@@ -1041,6 +1041,24 @@ def scale(self, scaler, fit=True):
         self.scaler = scaler
         return self
 
+    def scale_numpy(self, scaler, fit=True):
+        '''
+        Scale the time series dataset using numpy.
+
+        :param scaler: sklearn scaler instance, StandardScaler, MaxAbsScaler,
+               MinMaxScaler and RobustScaler are supported.
+        :param fit: if we need to fit the scaler. Typically, the value should
+               be set to True for training set, while False for validation and
+               test set. The value is defaulted to True.
+        '''
+        col_list = self.target_col + self.feature_col
+        if fit:
+            scaler.fit(self.df[col_list])
+        scaled_data_numpy = scale_timeseries_numpy(self.df[col_list].values, scaler)
+        self.df[col_list] = pd.DataFrame(scaled_data_numpy, columns=col_list)
+        self.scaler = scaler
+        return self
+
```

Comment:
move this to `scale` method, we should directly call `scale_timeseries_numpy` for the cases where `fit=False`